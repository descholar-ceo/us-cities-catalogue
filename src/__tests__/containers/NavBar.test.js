import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../containers/NavBar';

describe('Test the snapshot of Navbar component', () => {
  it('Render the correct snapshot', done => {
    const NavbarSnapshot = renderer.create(<BrowserRouter><Navbar renderedComponent="details" /></BrowserRouter>).toJSON();
    expect(NavbarSnapshot).toMatchSnapshot();
    done();
  });
});
