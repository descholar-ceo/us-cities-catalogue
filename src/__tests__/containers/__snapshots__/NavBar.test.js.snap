// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the snapshot of Navbar component Render the correct snapshot 1`] = `
<div
  className="header"
>
  <h1>
    <a
      href="/"
      onClick={[Function]}
    >
      UsCities
    </a>
  </h1>
  
</div>
`;
